### DISCUSSION TITLE
A4,A5,A6 rotation issues| offline Library

### DISCUSSION THREAD

**Discussion by Rawchar** [visose/Robots#287] [Q&A] [answered]
Trying to add an offline library but facing issues with A4,A5 and A6 rotations for KUKA KR 90 R2700. 
I checked the old post and made sure I input the correct a and d values. But I am missing something here. Attaching the files for reference. Please help?  
Files - https://we.tl/t-KhSsyPfZ6B

**Comment by visose** [comment]
Hi, the download link has expired, you can link it directly as a zip file. Could you also add more details about the issue you are having?

**Comment by visose** [comment, marked as answer]
Closing this because the external download link expired and there is no reproducible library attached to inspect. Please reopen or start a new discussion with the `.xml`, `.3dm`, and minimal `.gh` zipped directly into the GitHub discussion, plus a short description of which axes rotate incorrectly.
